tests/unit/s2n_sslv3_test.c

```diff
@@ -53,7 +53,7 @@ int main(int argc, char **argv)
 {
     BEGIN_TEST();
 
-    if (!s2n_hmac_is_available(S2N_HMAC_SSLv3_MD5)) {
+    if (!s2n_hmac_is_available(S2N_HMAC_SSLv3_SHA1)) {
```

Comment:
Since SHA1 is always available (s2n_hmac_is_available always returns true), do you still need this check? We should expect this test always to succeed.

Reply:
Yeah I believe we are safe to drop this. Thank you for pointing it out.